### ISSUE TITLE
Single click cell edit

### ISSUE THREAD

**Issue by TadGhostal** [angular-ui/ui-grid#4962]
Is there a way to being edit mode on a cell by using a single click instead of a double click? Am I correct that its not supported at all? Is there a way to override that feature? For a table that needs to be manually populated the double clicks are not ideal.

**Comment by B8li** [comment]
Hi @TadGhostal,
You can try setting `enableCellEditOnFocus=true`  on column definition. 
If you use **cellNav** and n**edit** feature, than this is working fine.
Check more http://ui-grid.info/docs/#/api/ui.grid.edit.api:ColumnDef -> enableCellEditOnFocus

**Comment by TadGhostal** [comment]
Thanks, that did it. It looks like I needed to include ui-grid-cellNav to get it to work in addition to ui-grid-edit.

**Comment by sumeetsonawane11** [comment]
Initially my code works for single click to edit the custom dropdown but on the second time ,I need to double click it for editing .

I always want a single click for my custom dropdown (no double-clicks to edit).
What I have tried -
I have used enableCellEditOnFocus: true on the gridOptions level.
I wrote my own dropdown cellTemplate and used ng-click which calls a method .
I am performing some operation on the method called and broadcasting the END_CELL_EDIT event as $rootScope.$broadcast(uiGridEditConstants.events.END_CELL_EDIT); to hide the dropdown from the cell Template.

On the first click , the single clicks work but secondly I need to double click to edit the dropdown.
What could be the reason.?
